pkg/odo/cli/service/create.go

```diff
@@ -119,6 +120,11 @@ func (o *ServiceCreateOptions) validateServiceName(i interface{}) (err error) {
 	return
 }
 
+var validateNameFn = func(i interface{}) error {
```

Comment:
Why not use the normal function?

Reply:
I am bit confused here, can you please add comments here ?

Reply:
Indeed, let me check.

Reply:
Looks like rebasing errors… downside of having a PR opened for so long. :(

Reply:
Fixed.